config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -41,17 +41,16 @@ presubmits:
   - name: pull-release-test
     always_run: true
     decorate: true
-    path_alias: k8s.io/release
     extra_refs:
     - org: kubernetes
-      repo: test-infra
+      repo: release
       base_ref: master
-      path_alias: k8s.io/test-infra
+      path_alias: k8s.io/release
     spec:
       containers:
-      - image: launcher.gcr.io/google/bazel:0.29.1
+      - image: gcr.io/k8s-staging-release-test/releng-ci-bazel:latest
         command:
-        - ../test-infra/hack/bazel.sh
+        - bazel
```

Comment:
I'm not sure if we need the `command` because the `entrypoint` of the image is already `bazel`. Making it explicit should not hurt here, right?

Reply:
Yes! Explicit please.
I _think_ Prow does a thing where it requires an explicit command.

Reply:
pod-utils does not respect image entrypoints, you have to set `command` if `decorate` is set to `true`

Reply:
Thanks for the hint, this is good to know. :)